src/core/heating_cooling_loads.jl

```diff
@@ -139,8 +145,8 @@ struct SpaceHeatingLoad
         annual_mmbtu::Union{Real, Nothing} = nothing,
         monthly_mmbtu::Array{<:Real,1} = Real[],
         # addressable_load_fraction,  # TODO
-        fuel_loads_mmbtu_per_hour::Array{<:Real,1} = Real[],
-        time_steps_per_hour::Int = 1,
+        fuel_loads_mmbtu_per_hour::Array{<:Real,1} = Real[], # TODO: shouldn't this be per time_step? 
```

Comment:
@Bill-Becker are you ok with me changing `fuel_loads_mmbtu_per_hour` to `fuel_loads_mmbtu_per_time_step` for both `SpaceHeatingLoad` and `DomesticHotWaterLoad` ?

Reply:
@adfarth I think that makes sense to be consistent with `electric_load.loads_kw` which can have different `time_steps_per_hour`. Would we have to change anything else or is it already being processed effectively as "per_time_step"?

Reply:
I left an in-line comment, but I don't believe this should have the `* time_steps_per_hour` added.

Reply:
Undid this change in https://github.com/NREL/REopt.jl/pull/82/commits/ff447d6e42571aab42437d61c5b95a7a19c7e955